packages/mongodb-memory-server-core/src/util/postinstall-helper.ts

```diff
@@ -23,11 +30,22 @@ if (typeof envSystemBinary === 'string') {
   process.exit(0);
 }
 
-(async () => {
+export async function postInstall(version?: string, local?: boolean): Promise<never | void> {
```

Comment:
Maybe `downloadBinaryToGlobal`? I dislike this name but it even better than `postInstall`.

Reply:
`downloadBinaryToGlobal` wouldn't really apply, because it is not always downloading to global (parameter `local`)

Reply:
Maybe `postInstallDownload`, `postInstallEnsureBinary`?
This method should have some information about what it does. Just `postInstall` name is too general.

Anyway, you may keep your current method name if you dislike any names above and don't have any alternatives

Reply:
using `postInstallEnsureBinary` in aca826211351e73458c56e6d0dce99d068e05941